tests/python/dashboard/util.py

```diff
@@ -0,0 +1,22 @@
+def wait_for_ready(sb, timeout=60):
+    for i in range(timeout):
+        print(f"wait_for_ready {i}")
+        if sb.is_element_present(".trame__loader"):
+            sb.sleep(1)
+        else:
+            print("Ready")

```

Comment:
This is code directly from https://github.com/Kitware/trame-client/blob/master/trame_client/utils/testing.py. Not sure of the process to credit, if any.

Reply:
Add inline comments with links? :)